src/compose/service-manager.ts

```diff
@@ -59,12 +59,30 @@ const volatileState: Dictionary<Partial<Service>> = {};
 export const getAll = async (
 	extraLabelFilters: string | string[] = [],
 ): Promise<Service[]> => {
-	const filterLabels = ['supervised'].concat(extraLabelFilters);
-	const containers = await listWithBothLabels(filterLabels);
+	// From balenaOS v3 containers will be created from multiple locations
+	// (preloaded in the OS, start-resin-supervisor script). All of these
+	// containers will be required to provide an app-uuid so is a good time to
+	// phase out the 'supervised' label. On the next major release after the
+	// v3 supervisor, this code should be removed
```

Comment:
Images have just started adding the supervised label on creation. I say just started because I had to patch a bug from a device having old images without this label https://github.com/balena-os/balena-supervisor/pull/1608. Do we really want to start removing the supervised label from services ?